src/components/Chatting/ChatInput.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useCallback } from 'react';
+interface ChatInputProps {
+  setIsCaution: any;
+  setInputText: any;
+  inputText: string;
+  isActiveInput: boolean | undefined;
+  isCaution: boolean | undefined;
+}
+function ChatInput({
+  setIsCaution,
+  inputText,
+  setInputText,
+  isActiveInput,
+  isCaution,
+}: ChatInputProps) {
+  const textChange = (e: any) => {
```

Comment:
typescript 에서 any 타입을 쓰는걸 지양하는게 좋아서 타입을 명시해주면 좋을거 같습니다!
```suggestion
  const textChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```